docs/configuration_requirements.rst

```diff
@@ -0,0 +1,19 @@
+Configuration requirements
+**************************
+
+.. only:: iso
+
+    This page shows the configuration requirements for the Inmanta Service Orchestrator |version_major| version.
+    These constraints are defined for the whole lifetime of this major version i.e.
+    they are not pinned to a specific three-digit version.
```

Comment:
We should mention somewhere that there is a machine-consumable version of these tables at \<insert link>. I think we may even want to link to both the json and the plain requirements file.